model/src/main/kotlin/utils/PurlExtensions.kt

```diff
@@ -98,6 +97,8 @@ fun KnownProvenance.toPurlExtras(): PurlExtras =
                 subpath = vcsInfo.path
             )
         }
+
+        else -> PurlExtras()
```

Comment:
Better use `is UnknownProvenance` because then we get a compiler error in case a new provenance type is introduced (even though that's very unlikely).